nf_core/lint.py

```diff
@@ -802,6 +803,21 @@ def check_pipeline_todos(self):
                                 l = '{}..'.format(l[:50-len(fname)])
                             self.warned.append((10, "TODO string found in '{}': {}".format(fname,l)))
 
+    def check_pipeline_name(self):
+        """Check whether pipeline name adheres to lower case/no hyphen naming convention"""
+        
+        #Extract last part of given path, a.k.a. the workflow name
+        workflow_name = os.path.basename(os.path.normpath(self.path))
```

Comment:
This won't always work - for example, I call my forks of pipelines different things. But the good news is that we already have the pipeline name in the class for you to use 🎉 `self.pipeline_name`

Reply:
Thanks for the review 😃 . Added the changes